Src/BootCamp.Chapter/Computer/Body.cs

```diff
@@ -6,5 +6,15 @@ namespace BootCamp.Chapter.Computer
 {
     public class Body
     {
+        private string _manufacturersId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_manufacturersId' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)